components/observations/ObservationDetailView.tsx

```diff
@@ -216,34 +217,83 @@ export const ObservationCard: React.FunctionComponent<{
   }, [postHog, observation.center_id, observation.id]);
   useFocusEffect(recordAnalytics);
 
+// route.path will have the link we would put into the share button here
+const route = useRoute(); 
+
+// currently the back button will leave to the list of obs of your current default center
```

Comment:
It looks like we should be able to use a built-in hook [`useLinkBuilder()`](https://reactnavigation.org/docs/use-link-builder) to generate the link for this page - that would make sure we're not ever at risk of generating a link that would not be correct as per the linking configuration.

Reply:
agreed - the link generation can def use work but I think my emphasis was lets just see if the sharing works as we expected, then we will actually clean up all the easier stuff 😄 but happy to clean up some of this now, whatever you think makes the most sense

Reply:
Let's see if we can do this in our first pass?

Reply:
sounds good to me - I may have time later today or tomorrow, but if you have time sooner go ahead and make the change